packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -47,6 +47,16 @@ export class Tabs extends FASTElement {
      */
     @attr
     public activeid: string;
```

Comment:
I believe this should have an `@internal` tsdoc flag unless this is intended to be used by authors.

Reply:
Why would this be internal? Its an attribute authors can write to.